spec/controllers/development_metrics_controller_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'rails_helper'
 
 RSpec.describe DevelopmentMetricsController, type: :controller do
```

Comment:
maybe later we could start to use request instead, im not sure but i think @santib sayd it is gonna be deprecated